translator-string/src/org/jetbrains/dukat/translatorString/StringTranslator.kt

```diff
@@ -179,14 +179,14 @@ private fun translateLambdaParameters(parameters: List<LambdaParameterModel>): S
 fun TypeModel.translate(): String {
     return when (this) {
         is TypeParameterReferenceModel -> {
-            val res = mutableListOf(name.translate())
+            val res = mutableListOf(name.translate(false))
             if (nullable) {
                 res.add("?")
             }
             res.joinToString("")
         }
         is TypeValueModel -> {
-            val res = mutableListOf(value.translate())
+            val res = mutableListOf(value.translate(false))
```

Comment:
For the future: when boolea is used as method parameter, better to use named parameter for discoverability, because from github interface very hard to uderstad, what `false` means in context.
Ca be used `shortNameForKotlin = false`